packages/core/src/transforms/__tests__/resize.spec.ts

```diff
@@ -545,6 +523,7 @@ describe('withoutReduction', () => {
       //@ts-expect-error we know this is safe
       const { image } = await applyTransforms([resize({ withoutReduction: 'true', width: '300;900' }, dirCtx)], img)
 
+      expect(dirCtx.warn).toHaveBeenCalledOnce()
```

Comment:
Not sure we want these assertions in the existing tests, but they have been added in one (out of two) tests where there was a warning, and in one test where there was no warning.